src/maneuvers/CMakeLists.txt

```diff
@@ -12,3 +15,32 @@ target_link_libraries(mission
     dronecode_sdk_telemetry
     dronecode_sdk_mission
    )
+
+##########
+# RTL_testing
+##########
+
+add_executable(RTL_testing
+    RTL_testing.cpp)
+
+include_directories(
+    px4maneuvers/src/external/Dronecode_sdk/core
```

Comment:
this line should not be necessary. core should be included automatically with any of the droncecode libs